lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,744 @@
+description:
+- Allows to create, drop or change the definition of a sequence generator
+  U(https://www.postgresql.org/docs/current/sql-createsequence.html).
+version_added: '2.8'
+options:
+  sequence:
+    description:
+    - The name (optionally schema-qualified) of the sequence.
+    required: true
+    type: str
+    aliases:
+    - name
+  state:
+    description:
+    - The sequence state.
+    - I(state=absent) is mutually exclusive with I(data_type), I(increment), I(minvalue), I(maxvalue),
+      I(start), I(cache), I(cycle), I(rename_to) , I(newschema) and I(owner).
+    default: present
+    choices: [ absent, present ]
+    type: str
+  data_type:
+    description:
+    - Specifies the data type of the sequence. Valid types are bigint, integer, and smallint. bigint is the default.
+      The data type determines the default minimum and maximum values of the sequence.
+    - Supported from PostgreSQL 10.
+    required: false
+    choices: [ bigint, integer, smallint ]
+    type: str
+  increment:
+    description:
+    - Increment specifies which value is added to the current sequence value to create a new value.
+    - A positive value will make an ascending sequence, a negative one a descending sequence. The default value is 1.
+    required: false
+    type: int
+  minvalue:
+    description:
+    - Minvalue determines the minimum value a sequence can generate. The default for an ascending sequence is 1.
+      The default for a descending sequence is the minimum value of the data type.
+    required: false
+    type: int
+  maxvalue:
+    description:
+    - Maxvalue determines the maximum value for the sequence. The default for an ascending sequence is the maximum
+      value of the data type. The default for a descending sequence is -1.
+    required: false
+    type: int
+  start:
+    description:
+    - Start allows the sequence to begin anywhere. The default starting value is
+      I(minvalue) for ascending sequences and I(maxvalue) for descending ones.
+    required: false
+    type: int
+  cache:
+    description:
+    - Cache specifies how many sequence numbers are to be preallocated and
+      stored in memory for faster access. The minimum value is 1 (only one
+      value can be generated at a time, i.e., no cache), and this is also
+      the default.
+    required: false
+    type: int
+  cycle:
+    description:
+    - The cycle option allows the sequence to wrap around when the I(maxvalue) or I(minvalue) has been reached by an
+      ascending or descending sequence respectively. If the limit is reached, the next number generated will be the
+      minvalue or maxvalue, respectively.
+    - If false (NO CYCLE) is specified, any calls to nextval after the sequence has reached its maximum value will
+      return an error. False (NO CYCLE) is the default.
+    required: false
+    type: bool
+  cascade:
+    description:
+    - Automatically drop objects that depend on the sequence, and in turn all
```

Comment:
Better to define what the default is

Reply:
I like it to not explicit define the default (because the SQL command will increase in that case and it is not necessary) and just say that this is the "implicit" default

Reply:
I changed it to only have the option `CASCADE`. I removed the implicit default option `RESTRICT`.